ui/lib/core/addon/components/info-table-item-array.hbs

```diff
@@ -1,51 +1,72 @@
-<div data-test-info-table-item-array {{did-insert (perform this.fetchOptions)}}>
+{{! the class linkable-item is needed for the read-more component }}
```

Comment:
Claire, I played with the styling a lot, and this was the only solution I could come up with to get the styling to work. It works great and is only adding one class, but you'll have more background on the ReadMore component and styling, so if I'm missing something simple, let me know. I tried all sorts of max-widths, etc on parents and it didn't see to work.